decidim-core/app/presenters/decidim/menu_presenter.rb

```diff
@@ -36,7 +36,7 @@ def render
       end
     end
 
-    private
+    protected
```

Comment:
No need to change visibility here. `protected` methods have very few use cases in Ruby and I don't think this is one...

Reply:
Yeah, I know. However, it doesn't make any difference. In general, I avoid `protected` methods unless I need them, and I've never yet needed them :).

Reply:
However, I'm checking other parts of the code, and I see that `protected` is used similarly in other parts of the code: to "document" that the methods are being overriden or extended by subclasses. I don't fancy this style, but if the others want to keep it, this change makes sense for consistency.

Reply:
I've fixed all your comments but the last one. I'm open to change it by whatever you consider is better.

Reply:
Thanks @jsperezg! Sorry for being a PITA here, it's not very important and I just saw @josepjaume's message "this is the last change I ask for" :laughing:. We can leave it like it is and maybe we can review our global usage of the `protected` keyword another time. :+1:

Reply:
No worries @deivid-rodriguez , I'm always open to learn a new point of view that helps me to make better code.